res/themes/unthemed/client/client.css

```
@@ -0,0 +1,794 @@
+Notifications::custom {
+    secondary-color: silver;
+    mods-color: red;
+}
+
+PlayerColors::custom {
+  you: red;
+  foe: crimson;
+  foe_mod: lightslategrey;
+  foe_chatterbox: darkred;
+  friend: lightskyblue;
+  friend_mod: medium orchid;
```

Biome (2.1.2)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)